packages/sources/src/sourceFormRenderer/components/CardSelect.js

```diff
@@ -64,12 +65,12 @@ class CardSelect extends React.Component {
                     isCompact={ true }
                 >
                     <div className={ disabled ? 'disabled' : '' }>
-                        <CardHeader>
+                        <CardHeader className={Component ? 'text-elipsis' : '' }>
```

Comment:
Just a tip. If you have more conditional classnames in this app, consider using library like https://www.npmjs.com/package/clsx